src/StyleCI/Fixers.php

```diff
@@ -373,37 +382,39 @@ class Fixers
         'whitespacy_lines',
     );
 
-
     /**
      * @var string[]
+     *
+     * @deprecated
      */
     public static $aliases = array(
         'phpdoc_params' => 'phpdoc_align',
         'join_function' => 'alias_functions',
     );
 
-
     /**
      * @var string[]
+     *
+     * @deprecated
      */
     public static $conflicts = array(
-        'concat_with_spaces' => 'concat_without_spaces',
-        'long_array_syntax' => 'short_array_syntax',
+        'concat_with_spaces'              => 'concat_without_spaces',
+        'long_array_syntax'               => 'short_array_syntax',
         'no_blank_lines_before_namespace' => 'single_blank_line_before_namespace',
-        'phpdoc_var_to_type' => 'phpdoc_type_to_var',
-        'psr0' => 'psr4',
-        'unalign_double_arrow' => 'align_double_arrow',
-        'unalign_equals' => 'align_equals',
+        'phpdoc_var_to_type'              => 'phpdoc_type_to_var',
+        'psr0'                            => 'psr4',
+        'unalign_double_arrow'            => 'align_double_arrow',
+        'unalign_equals'                  => 'align_equals',
     );
 
     public static function getPresets()
```

Comment:
I can't include https://github.com/StyleCI/Fixer package because it require your custom php-cs-fixer which have same namespaces as the official... that give me conflicts.